fs/xfstests.py

```diff
@@ -119,8 +119,8 @@ def setup_nvdimm(self):
                     self.plib.destroy_namespace(region=self.region, force=True)
                 self.plib.destroy_namespace(
                     region=self.region_ldev, force=True)
-                # XFS restrict max log size to 2136997888
-                logdev_size = min(logdev_size, 2136997888)
+                # XFS restrict max log size to 2130706432
```

Comment:
here as i will say it can handle via YAML input so we need not change code every time as we can make code more robust

Reply:
this cannot be yaml input since it is dynamically computed from the existing pmem device size and cannot exceed 2G mark.

Reply:
@harish-24 would be readable if you can expand the value either in comment or in the code.

Reply:
@narasimhan-v Pushed with comment expanded. Thanks.